amd/src/mention_users.js

```diff
@@ -91,6 +89,21 @@ define(['jquery', 'core/ajax', 'local_mention_users/tribute'], function($, ajax)
         }]
       })
 
+        window.tributeinstance = tribute;
+        window.usersarray = users_array;
+
+      let user = null;
+      let userid = window.location.search.match(/u=(\d+)/);
+      let useridpassed = false;
+      let windowhashash = false;
+      if (userid !== null) {
+        // This is a bit hacky, but if a user has the auto-tag functionality, once they submit, it retags user, so if
+        // there is the hash (which is added to scroll to the post), don't allow aut-tagging
+        useridpassed = true;
+        windowhashash = window.location.hash !== '';
+        user = users_array.filter(function(item) {return item.value == userid[1]})[0];
```

Comment:
es6 suggestion to note, this can be shortened to:
`users_array.filter((item) => item.value == userid[1])[0];`